packages/openneuro-server/src/handlers/config.js

```diff
@@ -35,7 +35,9 @@ const config = {
   },
 
   analytics: {
-    trackingId: process.env.GOOGLE_TRACKING_ID,
+    trackingIds: process.env.GOOGLE_TRACKING_IDS.split(',').map(id =>
+      id.trim(),
+    ),
```

Comment:
This should handle the case where the value is not provided or only one value is provided.

Reply:
One value will just result in an array containing that one value. No value will result in an array with an empty string as the first value

Reply:
Okay, the issue I was having with this is a setup problem with my config for this, not an issue with this section. Thanks!